src/main/java/org/fenixedu/academic/servlet/FenixInitializer.java

```diff
@@ -203,4 +215,85 @@ private void initializeExecutionDegreeLessonPeriods() {
+                EducationLevelType educationLevelType =
+                        EducationLevelType.create(code, name, true, schoolLevelType.isForStudent(),
+                                schoolLevelType.isForStudentHousehold(), schoolLevelType.isForMobilityStudent(),
+                                schoolLevelType.isOther(), schoolLevelType.isPhDDegree(),
+                                schoolLevelType.isSchoolLevelBasicCycle(), schoolLevelType.isHighSchoolOrEquivalent(),
+                                schoolLevelType.isHigherEducation());
+
+                schoolLevelType.getEquivalentDegreeClassifications().forEach(c -> {
+                    DegreeClassification classification = DegreeClassification.readByCode(c);
+
+                    if (classification != null) {
+                        educationLevelType.addDegreeClassifications(classification);
+                    }
+                });
+            }
+        }
+
+        Log.warn("Finished population of Education Level Types. Instances created: " + EducationLevelType.findAll().count());
+        Log.warn("---------------------------------------");
+    }
+
+    @Atomic(mode = TxMode.WRITE)
+    private void initializeProfessionCategoryTypes() {
+        if (ProfessionCategoryType.findAll().findAny().isPresent()) {
+            return;
+        }
+
+        Log.warn("---------------------------------------");
+        Log.warn("Starting population of Profession Category Types");
+
+        for (org.fenixedu.academic.domain.ProfessionType professionType : org.fenixedu.academic.domain.ProfessionType.values()) {
+            String code = professionType.getName();
+            LocalizedString name = BundleUtil.getLocalizedString(Bundle.ENUMERATION, professionType.getQualifiedName());
+
+            if (ProfessionCategoryType.findByCode(code).isEmpty()) {
+                ProfessionCategoryType.create(code, name, true);
+            }
+        }
+
+        Log.warn("Finished population of Profession Category Types. Instances created: " + ProfessionCategoryType.findAll()
+                .count());
+        Log.warn("---------------------------------------");
+    }
+
+    @Atomic(mode = TxMode.WRITE)
+    private void initializeProfessionalStatusType() {
+        if (ProfessionalStatusType.findAll().findAny().isPresent()) {
+            return;
+        }
+
+        Log.warn("---------------------------------------");
+        Log.warn("Starting population of Professional Status Types");
+
+        for (ProfessionalSituationConditionType professionalSituation : ProfessionalSituationConditionType.values()) {
+            String code = professionalSituation.getName();
+            LocalizedString name = BundleUtil.getLocalizedString(Bundle.ENUMERATION, professionalSituation.getQualifiedName());
+
+            if (ProfessionalStatusType.findByCode(code).isEmpty()) {
+                ProfessionalStatusType.create(code, name, true);
```

Comment:
Beware, no all of the `ProfessionalSituationConditionType` enum values are active